packages/core/docker/Dockerfile

```diff
@@ -1,67 +1,26 @@
-FROM node:20-alpine AS base
+FROM latitudedata/base AS installer
 
-ARG DATABASE_URL
 ARG NODE_ENV=development
 
-# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
-RUN apk update
-RUN apk add --no-cache libc6-compat
-
-# Install pnpm
-ENV PNPM_HOME="/pnpm"
-ENV PATH="$PNPM_HOME:$PATH"
-RUN corepack enable && corepack prepare pnpm@8.6.3 --activate
-
-# Install Turbo
-RUN pnpm i -g turbo
-
-FROM base AS builder
-
-WORKDIR /app
-
-COPY . .
 RUN turbo prune @latitude-data/core --docker
 
-FROM base AS installer
-
-ARG NODE_ENV
-
-WORKDIR /app
+FROM installer AS runner
```

Comment:
Bit weird that the `latitude/base` stage is named `installer`, and the actual `installer` stage is named `runner`

Reply:
The base is the nodejs and pnpm installer I guess is for that

Reply:
hmm you have to think it as "FROM _this image_ START A NEW IMAGE WITH NAME _installer_"